example/lib/isolate.dart

```diff
@@ -91,19 +91,27 @@ class _MyHomePageState extends State<MyHomePage> {
     maxLines: 10000,
   );
 
-  @override
-  void initState() {
-    super.initState();
-    terminal.start();
+  Future<TerminalIsolate> _ensureTerminalStarted() async {
+    if (!terminal.isReady) {
+      await terminal.start();
+    }
+    return terminal;
   }
 
   void onInput(String input) {}
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: SafeArea(
-        child: TerminalView(terminal: terminal),
+      body: FutureBuilder(
+        future: _ensureTerminalStarted(),
+        builder: (context, snapshot) {
+          return SafeArea(
+            child: snapshot.hasData
+                ? TerminalView(terminal: snapshot.data as TerminalIsolate)
+                : Text('Initializing...'),
```

Comment:
Would it be better to place an empty TerminalView here instead of a Text?

Reply:
yes, good point.
I would add a Container that gets shown during that time that has the same background and opacity?
Because I think it would harm the TerminalView to also support giving it no Terminal?